### ISSUE TITLE
YAS Docker

### ISSUE THREAD

**Issue by Adminius** [codealchemist/youtube-audio-server#15]
Hi, thanks for your work! it's really greate!
Is it planned to publish YAS in a docker containter for it? (all in one with all dependencies)

**Comment by codealchemist** [comment]
Hi @Adminius!
Just published to docker cloud:
https://hub.docker.com/r/codealchemist/youtube-audio-server/

Please, let me know if it works for you.
Thx!

**Comment by Adminius** [comment]
Hi, thx, but it doesn't work because of missing ffmpeg.

```

Status: Downloaded newer image for codealchemist/youtube-audio-server:latest
TEMP AUDIO FOLDER: /usr/local/lib/node_modules/youtube-audio-server/temp-audio
                    _         _
  _   _  ___  _   _| |_ _   _| |__   ___
 | | | |/ _ \| | | | __| | | | '_ \ / _ \
 | |_| | (_) | |_| | |_| |_| | |_) |  __/
  \__, |\___/ \__,_|\__|\__,_|_.__/ \___|
  |___/

                  _ _
   __ _ _   _  __| (_) ___        ___  ___ _ ____   _____ _ __
  / _` | | | |/ _` | |/ _ \ _____/ __|/ _ \ '__\ \ / / _ \ '__|
 | (_| | |_| | (_| | | (_) |_____\__ \  __/ |   \ V /  __/ |
  \__,_|\__,_|\__,_|_|\___/      |___/\___|_|    \_/ \___|_|

 🔈  YOUTUBE AUDIO SERVER listening on http://localhost:80!
--------------------------------------------------------------------------------
events.js:167
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find ffmpeg
    at /usr/local/lib/node_modules/youtube-audio-server/node_modules/fluent-ffmpeg/lib/processor.js:136:22
    at /usr/local/lib/node_modules/youtube-audio-server/node_modules/fluent-ffmpeg/lib/capabilities.js:123:9
    at /usr/local/lib/node_modules/youtube-audio-server/node_modules/async/dist/async.js:473:16
    at next (/usr/local/lib/node_modules/youtube-audio-server/node_modules/async/dist/async.js:5329:29)
    at /usr/local/lib/node_modules/youtube-audio-server/node_modules/async/dist/async.js:969:16
    at /usr/local/lib/node_modules/youtube-audio-server/node_modules/fluent-ffmpeg/lib/capabilities.js:116:11
    at /usr/local/lib/node_modules/youtube-audio-server/node_modules/fluent-ffmpeg/lib/utils.js:223:16
    at F (/usr/local/lib/node_modules/youtube-audio-server/node_modules/which/which.js:68:16)
    at E (/usr/local/lib/node_modules/youtube-audio-server/node_modules/which/which.js:80:29)
    at /usr/local/lib/node_modules/youtube-audio-server/node_modules/which/which.js:89:16
Emitted 'error' event at:
    at emitEnd (/usr/local/lib/node_modules/youtube-audio-server/node_modules/fluent-ffmpeg/lib/processor.js:424:16)
    at /usr/local/lib/node_modules/youtube-audio-server/node_modules/fluent-ffmpeg/lib/processor.js:433:16
    at /usr/local/lib/node_modules/youtube-audio-server/node_modules/async/dist/async.js:473:16
    [... lines matching original stack trace ...]
    at /usr/local/lib/node_modules/youtube-audio-server/node_modules/async/dist/async.js:473:16
    at next (/usr/local/lib/node_modules/youtube-audio-server/node_modules/async/dist/async.js:5329:29)
    at /usr/local/lib/node_modules/youtube-audio-server/node_modules/async/dist/async.js:969:16
    at /usr/local/lib/node_modules/youtube-audio-server/node_modules/fluent-ffmpeg/lib/capabilities.js:519:16
    at /usr/local/lib/node_modules/youtube-audio-server/node_modules/fluent-ffmpeg/lib/processor.js:136:16
```

P.S. i started yas with different port, because :80 is already busy:

```
docker run \
  --name yas \
  -p 1234:80 \
  codealchemist/youtube-audio-server
```

how can i expose path for media downloads to host file system?

**Comment by codealchemist** [comment]
Fixed in #19.
Please, let me know if you can verify.
Thanks!

About exposing the path... I think you could mount it or maybe launch a static web server to serve the files. Maybe you can add that as a feature request. I think it could be useful.

Cheers!

**Comment by Adminius** [comment]
thx. it works perfect:

```
docker run -d \
  --name yas \
  -p 1234:80 \
  codealchemist/youtube-audio-server
```
